internal/wgnl/configure_linux_test.go

```diff
@@ -146,7 +147,7 @@ func TestLinuxClientConfigureDevice(t *testing.T) {
 											0x00, 0x00, 0x00, 0x00,
 											0x00, 0x00, 0x00, 0x33,
 										},
-										Port: 51820,
+										Port: binary.BigEndian.Uint16(nlenc.Uint16Bytes(51820)),
```

Comment:
Sorry; can you use the helper in the tests too?  They're in the same package so should be fine.